book/modules/appendix1-exercises.tex

```diff
@@ -1,7 +1,57 @@
-
 \begin{exercises}
 
 	\begin{problist}
-		\prob XXX Finish
+		\prob For each equation given below, determine if it is linear. If so, write it in its standard form. If not, explain what makes it nonlinear.
```

Comment:
Run this file through https://siefkenj.github.io/latex-parser/ so that the lines are wrapped.

Reply:
linear -> linear equation.

Don't mention `standard form`. It's not important.

`\sec` and `\csc` and `\cot` should already be operators.